installing/cluster-capabilities.adoc

```diff
@@ -62,6 +62,12 @@ include::modules/operator-marketplace.adoc[leveloffset=+2]
 .Additional resources
 * xref:../operators/understanding/olm-rh-catalogs.adoc#olm-rh-catalogs[Red Hat-provided Operator catalogs]
 
+include::modules/node-tuning-operator-capability.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+* xref:../scalability-and-performance/using-node-tuning-operator.adoc[Using the Node Tuning Operator]
```

Comment:
It took me a second to catch this - folders are underscores (scalability_and_performance), files are dashes (using-node-tuning-operator.adoc). Additionally, we have to use anchors for file paths to work with our different distribution tools. I believe it's the customer portal, specifically, that requires anchors.

`* xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[Using the Node Tuning Operator]`